python-main.js

```diff
@@ -1225,9 +1228,8 @@ function web_editor(config) {
             // Push binary to board
             p = PartialFlashing.connectDapAsync()
                 .then(function() {
-                    var output = generateFullHexStr();
-                    var image = MemoryMap.fromHex(output).slicePad(0, PartialFlashingUtils.pageSize * PartialFlashingUtils.numPages);
-
+                    var output = generateFullHex("bytes");
+                    var image = micropythonFs.getIntelHexBytes(); //.slicePad(0, PartialFlashingUtils.pageSize * PartialFlashingUtils.numPages);
```

Comment:
This line can be removed, `output` and `image` should be the same at this point.